prober/dns_test.go

```diff
@@ -226,27 +230,23 @@ func TestAuthoritativeDNSResponse(t *testing.T) {
 	}{
 		{
 			config.DNSProbe{
-				IPProtocol: "ipv4",
-				QueryName:  "example.com",
+				QueryName: "example.com",
```

Comment:
Leave these as v4 please.

Reply:
with fallback?

Reply:
It doesn't matter either way.

Reply:
thats the problem, on my notebook some test failures if there is no fallback and it try only ipv4